src/sora-card.md

```diff
@@ -0,0 +1,71 @@
+# SORA Card
+
+## Key features of SORA Card
+
+SORA Card differentiates itself from other crypto cards and wallets by being a unique self-custodial crypto wallet with a DEX in the same app as your IBAN and debit card, while ensuring the highest privacy and security standards. SORA Card is specifically designed for the interoperable future by bridging multiple EVM based ecosystems and Polkadot’s Substrate networks into one solution.
+All of this will be made possible while it has the standard features you can expect from a debit card and banking wallet.
+
+_Now, what does all of this mean?_
+
+### What is a self-custodial wallet?
+
+SORA Card integrates with the self-custodial SORA Wallet and will later integrate with Fearless Wallet, the DeFI wallet for the Dotsama ecosystem. Other crypto cards like crypto.com and Binance Card are connected to custodial wallets.
+
+In a self-custodial solution, the customer retains full custody (possession) of their cryptoassets at all times. The wallet (SORA or Fearless Wallet) is merely an interface to conveniently manage your assets, and it connects directly to the blockchain without intermediaries. In the custodial solution, you trust the possession of your assets to the wallet provider and cannot transact directly on the blockchain.
+
+### How do I know if I’m using a self-custodial wallet?
+
+All self-custodial crypto wallets enable you—and only you—to possess the private key associated with your public address. This private key takes the form of either a file or a “mnemonic phrase” consisting of 12-24 randomly generated words. If your wallet does not have this option, it is a custodial wallet and you are not in full control of your crypto assets!
+
+## What is a DEX?
+
+Decentralized exchanges, also known as DEXs, are peer-to-peer marketplaces where cryptocurrency traders make transactions directly on-chain without handing over management of their funds to an intermediary or custodian. Polkaswap.io is a great example of a well-designed and efficient DEX. Executing transactions directly with a blockchain makes DEXs the ultimate in financial freedom and privacy. These transactions are facilitated through the use of self-executing agreements written in code called smart contracts.
```

Comment:
`makes DEXs the ultimate in financial freedom and privacy` the ultimate what? solution?

Reply:
Addressed